www/front_src/src/redux/reducers/tooltipReducer.js

```diff
@@ -0,0 +1,19 @@
+import * as actions from '../actions/tooltipActions';
+
+const initialState = {
+  toggled: false,
+  label: "",
+  x: 0,
+  y: 0
+}
+
+const tooltipReducer = (state = initialState, action) => {
+  if (action.type === actions.UPDATE_TOOLTIP) {
+    const {data} = action;
+    return {...state, ...data};
```

Comment:
spread operator before data is useless ;)

Reply:
it doesn't works without spread operator before data